jwql/tests/test_utils.py

```diff
@@ -344,3 +345,46 @@ def test_filesystem_path():
     location = os.path.join(get_config()['filesystem'], 'jw96003', filename)
 
     assert check == location
+
+
+def test_validate_config():
+    """Test that the config validator works.
+    """
+    # Make sure a bad config raises an error
+    bad_config_dict = {"just": "one_key"}
+
+    with pytest.raises(Exception) as excinfo:
+        _validate_config(bad_config_dict)
+    assert 'Provided config.json does not match the required JSON schema' \
+           in str(excinfo.value), \
+        'Failed to reject incorrect JSON dict.'
+
+    # Make sure a good config does not!
+    good_config_dict = {
+        "connection_string": "",
+        "database": {
```

Comment:
Good call on splitting this all into separate keys; it will be much easier to refer to individual components this way